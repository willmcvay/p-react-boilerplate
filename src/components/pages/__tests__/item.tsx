import * as React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Item from '../item'

describe('Item', () => {
  it('should match a snapshot', () => {
    expect(toJson(shallow(<Item />))).toMatchSnapshot()
  })
})
