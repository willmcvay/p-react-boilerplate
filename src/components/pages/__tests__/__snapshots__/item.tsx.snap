// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item should match a snapshot 1`] = `
<div>
  Item Page
</div>
`;
